yb-voyager/cmd/importData.go

```diff
@@ -183,7 +183,7 @@ func importData(importFileTasks []*ImportFileTask) {
 		}
 		time.Sleep(time.Second * 2)
 	}
-	executePostImportDataSqls()
+	// executePostImportDataSqls()
```

Comment:
instead of commenting, run it only for the yugabytedb dbtype for now and a TODO to implement it with Interface